### PR TITLE
add linkerd-certs to bundle

### PR DESCRIPTION
**Pull request by mleisa**
This PR: Add ´linkerd-certs-app´ to bundle

### Testing

- [ ] Fresh install works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM
- [ ] Upgrade from previous version works
  - [ ] AWS
  - [ ] Azure
  - [ ] KVM

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: helm/service-mesh-bundle/values.yaml [hunk 1]
```diff
@@ -90,3 +90,16 @@ apps:
     version: 0.11.0
     userConfig: {}
     extraConfigs: []
+  linkerd-certs:
+    appName: linkerd-certs
+    chartName: linkerd-certs
+    catalog: giantswarm-playground-test
+    enabled: true
```
Comment (mcharriere) [inline helm/service-mesh-bundle/values.yaml:97]: let's keep it disabled by default for now. you can enable on our tests with a values configMap

Comment (mleisa) [reply helm/service-mesh-bundle/values.yaml]: done

Comment (Gacko) [reply helm/service-mesh-bundle/values.yaml]: Can we get a real release of the `linked-certs` app in the `giantswarm-playground` catalog? :) At least I would not rely on the `-test` catalog for references, especially if we are releasing a new version of the bundle soon.

**Comment by Gacko** [comment]
LGTM!

**Review by Gacko — CHANGES_REQUESTED** [review]
Can we get a real release of the `linked-certs` app in the `giantswarm-playground` catalog? 🙂 At least I would not rely on the `-test` catalog for references, especially if we are releasing a new version of the bundle soon.

**Comment by mcharriere** [comment]
> Can we get a real release of the linked-certs app in the giantswarm-playground catalog? 🙂 At least I would not rely on the -test catalog for references, especially if we are releasing a new version of the bundle soon.

I agree. we should aim to merge this PR only after having our first release of `linkerd-certs`

**Comment by mleisa** [comment]
Reverting to draft until then

**Pull request closed without merging**